src/data/faq.json

```diff
@@ -1848,6 +1848,19 @@
                         "id": "ios",
                         "indexed": true,
                         "accordion": [
+                            {
+                                "title": "[Apple/iOS]: What are the requirements for installing the Corona-Warn-App on an iPhone?",
+                                "anchor": "ios_install_requirements",
+                                "active": false,
+                                "textblock": [
+                                    "If you want to use the Corona-Warn App, at least <b>iOS [ios.minimum-required-os-version]</b> must be installed on your iPhone. Thus, the Corona-Warn App is only available on devices which support at least this iOS version. To check which iOS version you are using, please follow the procedure described here: <a href='https://support.apple.com/en-us/HT201685' target='_blank' rel='noopener noreferrer'>Find the software version on your iPhone, iPad, or iPod (Apple Support)</a>.",

```

Comment:
Write "Corona-Warn-App" not "Corona-Warn App" when referring to the app. (Multiple places).

Reply:
```suggestion
                                    "If you want to use the Corona-Warn-App, at least <b>iOS [ios.minimum-required-os-version]</b> must be installed on your iPhone. Thus, the Corona-Warn-App is only available on devices which support at least this iOS version. To check which iOS version you are using, please follow the procedure described here: <a href='https://support.apple.com/en-us/HT201685' target='_blank' rel='noopener noreferrer'>Find the software version on your iPhone, iPad, or iPod (Apple Support)</a>.",
```

Reply:
Fixed with [`c8638dd`](https://github.com/corona-warn-app/cwa-website/pull/2738/commits/c8638ddf17e9100f98c8c61b62384ff00e69adcc).